ocaml/idl/datamodel.ml

```diff
@@ -1080,6 +1080,8 @@ let _ =
     ~doc:"The SR has no attached PBDs" ();
   error Api_errors.sr_full ["requested";"maximum"]
     ~doc:"The SR is full. Requested new size exceeds the maximum size" ();
+  error Api_errors.sr_source_space_insufficient [ ]
```

Comment:
I think it's better to add `SR` as the parameter of this exception

Reply:
I agree: add a parameter for the SR ref, so that it is easy to see which SR is full (a VM may have disks on multiple SRs).